rpc/common/moc_common_nodeinfo.proto

```diff
@@ -14,7 +14,11 @@ message Processor {
 	string speed = 3;
 	ProcessorType type = 4;
 	bool virtualization = 5;
-	uint32 logicalprocessors = 6;
+	bool hypervisorpresent = 6;
+	uint32 logicalprocessors = 7;
```

Comment:
You can't change the position of "logicalprocessors" without doing a breaking change. Can "logicalprocessors" be moved back to position 6?

Reply:
Yes, I have reverted it back to 6